docs/components/tasklist/userguide/using-tasklist.md

```diff
@@ -12,6 +12,8 @@ User tasks need an interaction from the user. This can be updating, adding varia
 
 If the user claimed a task, the task can be completed. Different task status filters help the user choose the desired task.
 
+Please note that when a user is granted tasklist access, the user will have full access to respective process instance data.
```

Comment:
I'd recommend noting this as an admonition. For example (with a few technical adjustments):
:::note
When a user is granted Tasklist access, the user has full access to the respective process instance data.
:::

Reply:
Done :+1: